llvm/test/CodeGen/SPIRV/pointers/typeof-ptr-int.ll

```diff
@@ -0,0 +1,29 @@
+; This test is to check that two functions have different SPIR-V type
```

Comment:
Thank you for tackling this one! This was a big (repeating) issue appearing when validating OpenCL CTS kernels. Surprisingly, the same function type definitions did not cause any runtime problems at least in OpenCL CTS.